osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/GetOutagesRequestList.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
```

Comment:
Looks like this should mention Alliander N.V. instead of Smart Society Services B.V.